index.html

```diff
@@ -366,9 +362,187 @@ <h4 class="card-list__caption">Check In & Check Out</h4>
         </div>
       </div>
     </section>
-    <section class="container-fluid section section--newsletter content-align-center">
+    <section class="container-fluid section section--visual">
```

Comment:
You don't need container-fluid as a wrapper for container. Please, remove and check all the document.